src/Parallel/Main.hpp

```diff
@@ -941,39 +942,37 @@ void Main<Metavariables>::post_deadlock_analysis_termination() {
 
 template <typename Metavariables>
 std::tuple<std::string, std::string, size_t>
-Main<Metavariables>::checkpoint_dir_basename_pad() const {
-  const std::string basename = "SpectreCheckpoint";
-  constexpr size_t pad = 6;
+Main<Metavariables>::checkpoints_dir_prefix_pad() const {
+  const std::string checkpoints_dir = "Checkpoints";
+  const std::string prefix = "Checkpoint";
+  constexpr size_t pad = 3;
```

Comment:
Yea they're cheap but become hard to read/count at some point. Try this: just glancing at `0000` you immediately know that it's 4 zeros. But glancing at `00000` you don't, you have to actually count. So I can make it 4 characters if you think we need that many, but anything beyond that I'd like to avoid. (also keep in mind that the checkpoint/segment names are actually not set in stone, I want to make sure that none of the code really relies on a hard-coded directory structure but just takes as input the files that it needs).

Reply:
OK, but why do you want to count the digits?  I don't see a reason that should be a design goal.

If you don't want to discuss the counter format in this PR then don't change it.

Reply:
I think readable directory names are a design goal. No need for excessive digits in my opinion. More important for segments than for checkpoints, so if you prefer I can keep the checkpoint format unchanged in this PR.

Reply:
I hadn't actually compared them, so I tried creating a list of `CheckpointNNN` and `CheckpointNNNNNN` names from 0 to 19 (I got tired of typing at that point).  I actually found the six-digit version to be significantly easier to read, because the string of zeros acts as a separator between "Checkpoint" and the number.  It makes it easier to visually locate the first nonzero digit.  Presumably the same would hold for other prefixes.

[Continuing from a thread below, but consolidating because the topic is the same]

> Three digits give us over 3 years in daily segments. That seems enough for the vast majority of simulations. As mentioned above the number of digits is not set in stone, so if we want to do a 5-year simulation or do hourly segments for some reason or something like that we can always increase the number of digits for that case.

You assume that segments will always be based on a timer.  This has not always been true for SpEC, and while it is a goal for SpECTRE I don't think we should assume that we will always achieve it immediately.  SpEC segments could be extremely short (minutes, maybe less) because they were based on some event in the simulation that forced a new segment.

Anyway, @markscheel, you're probably the most experienced SpEC-runner here.  Do you want to weigh in?